apps/nextjs/src/app/aila/page-contents.tsx

```diff
@@ -2,41 +2,17 @@
 
 import React from "react";
 
-import { Moderation } from "@oakai/db";
-import { type Message } from "ai/react";
-
 import { Chat } from "@/components/AppComponents/Chat/Chat/chat";
 import Layout from "@/components/AppComponents/Layout";
 import { ChatProvider } from "@/components/ContextProviders/ChatProvider";
 import LessonPlanTrackingProvider from "@/lib/analytics/lessonPlanTrackingContext";
 
-const ChatPageContents = ({
-  id,
-  isShared,
-  startingMessage,
-  initialMessages,
-  initialLessonPlan,
-  initialModerations,
-}: {
-  startingMessage?: string;
-  initialMessages?: Message[];
-  initialLessonPlan?: object;
-  id: string;
-  isShared: boolean | undefined;
-  initialModerations: Moderation[];
-}) => {
+const ChatPageContents = ({ id }: { id: string }) => {
   return (
     <Layout>
       <LessonPlanTrackingProvider chatId={id}>
-        <ChatProvider
-          key={`chat-${id}`}
-          id={id}
-          initialMessages={initialMessages}
-          initialLessonPlan={initialLessonPlan}
-          initialModerations={initialModerations}
-          startingMessage={startingMessage}
-        >
-          <Chat isShared={isShared} />
+        <ChatProvider key={`chat-${id}`} id={id}>
+          <Chat />
```

Comment:
Chat now handles the TRPC calls to the server, so we don't need to pass these in at this level now.